ArduPlane/RC_Channel.cpp

```diff
@@ -281,6 +282,22 @@ void RC_Channel_Plane::do_aux_function(const aux_func_t ch_option, const AuxSwit
        do_aux_function_flare(ch_flag);
        break;
 
+   case AUX_FUNC::RETRACT_MOUNT:
```

Comment:
Shouldn't this be going up into the common library?

Should be able to use `AP::mount()` to get the mount.

Reply:
I just copied what Copter does...not sure what you want me to do here...

Reply:
> I just copied what Copter does...not sure what you want me to do here...

Generally we really, really don't like copying code.  It means you end up with twice the number of bugs.

What we're after is moving the code from ArduCopter's RC_Channel library into the common RC_Channel library.

You actually helped doing this once before:

```
commit 4a51de724d228ac06f503b8cb875b52f5c8d112c
Author: Henry Wurzburg <hwurzburg@yahoo.com>
Date:   Sat Nov 2 23:18:34 2019 -0500

    Copter: move support for ADSB Avoidance enable up to RC_Channel

commit 1238d4b3e6dd6ef3e596058366dba5fd0d736198
Author: Henry Wurzburg <hwurzburg@yahoo.com>
Date:   Sat Nov 2 23:14:39 2019 -0500

    RC_Channel: add ADSB Avoidance enable option to RC options for plane

commit e12a841eb579f20331144377195ea1a60fecbe17
Author: Peter Barker <pbarker@barker.dropbear.id.au>
Date:   Mon Nov 4 11:18:01 2019 +1100

    RC_Channel: move support for ADSB avoidance switch enable up
```

Reply:
oh that common...I thought you meant some AP_Common....np...can move it into RC_Channel...

Reply:
done